Nuget.Config

```diff
@@ -2,6 +2,5 @@
 <configuration>
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="proget" value="http://proget.infragistics.local:81/nuget/IgniteUINuGet/" />
```

Comment:
I think this would still be needed if you want to restore the dependencies locally via the app project. We could remove it, together with the whole app, if we add instead a powershell script or something that directly pulls the packages we want, like in the pipeline, and update the readme with the new steps.